docs/api/state.md

```
@@ -0,0 +1,296 @@
+            name=name,
+            created_at=datetime.utcnow()
+        )
+```
+
+### 5. Compose Complex States
+```python
+class UserAccount(State):
+    identity: UserIdentity
+    profile: UserProfile
+    preferences: UserPreferences
+```
```

markdownlint-cli2 (0.17.2)

Files should end with a single newline character

(MD047, single-trailing-newline)